rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/WorkerSinkTask.java

```diff
@@ -421,7 +421,7 @@ private void pullMessageFromQueues() throws InterruptedException {
             if (null != pullResult && pullResult.getPullStatus().equals(PullStatus.FOUND)) {
                 this.incPullTPS(entry.getKey().getTopic(), pullResult.getMsgFoundList().size());
                 messages = pullResult.getMsgFoundList();
-                connectStatsManager.incSinkRecordReadTotalNums();
+                connectStatsManager.incSinkRecordReadTotalNums(messages.size());
```

Comment:
The reason is the same as the source

Reply:
add SourceRecordPollTotalTimes represents the source poll times, SinkRecordReadTotalTimes represents the sink read times